lib/InputTest.ts

```diff
@@ -168,7 +215,10 @@ export class InputTest extends EventEmitter {
    * Volume levels generated from the audio source during the run time of the
    * test.
    */
-  private readonly _values: number[] = [];
+  private readonly _volumeStatistics: VolumeStatistics = {
```

Comment:
nit: If you agree to rename `VolumeStatistics` to `VolumeStats`, can we rename `_volumeStatistics` to `_volumeStats`? We use `FooStats` patter all over rtc-diagnostics and client